theano/gpuarray/tests/test_elemwise.py

```diff
@@ -197,7 +197,7 @@ def test_perform_nan(self):
 
     def setUp(self):
         super(test_GpuCAReduceCuda, self).setUp()
-        if get_context(test_ctx_name).kind != 'cuda':
+        if get_context(test_ctx_name).kind != b'cuda':
```

Comment:
Thanks for the explanation. But I still find that ugly in the Python side. Can we make an automatic conversion from Bytes to string?